index.html

```diff
@@ -11,11 +11,8 @@
     <!--mobile version-->
     <header>
       <nav class="navbar">
-        <a href="#hello"><h4>PEDRO MARQUES</h4></a>
+        <a href="#"><h4>PEDRO MARQUES</h4></a>
         <ul class="nav-menu">
```

Comment:
- Great job so far following the Figma design in your menu implementation. Just a few extra issues to iron out 👍 
- [ ] (OPTIONAL)The battery and time menu is not really necessary, it is present in the design only to show how the page will look on the mobile view so it would be good to remove that 😄 
- Extra menu:
![Annotation 2022-04-12 125722](https://user-images.githubusercontent.com/85113567/162957789-89a40349-8917-4be1-afb7-0348182b58d6.jpg)
 - [ ] (OPTIONAL) I can see that your hamburger menu turns into an `x` icon which closes the mobile menu. As a result of this, it would be best to first prevent the scroll of the page when the mobile menu is opened, then remove the extra `x` icon of the menu since it has no functionality. In that way, when the menu is opened the user will be able to close it with the `x` icon from the hamburger menu and the user should not be able to scroll the page so the `x` icon of the hamburger menu will always remain on top 👍 
 - Your design:
![Annotation 2022-04-12 125218](https://user-images.githubusercontent.com/85113567/162958188-8df27b87-7c19-4ba5-97c4-cf68e11ae51d.jpg)